generators/client-2/templates/src/main/webapp/app/admin/user-management/_user-management-delete-dialog.html

```diff
@@ -1,15 +1,15 @@
-<form name="deleteForm" ng-submit="vm.confirmDelete(vm.user.login)">
+<form name="deleteForm" (ngSubmit)="confirmDelete(user.login)">
     <div class="modal-header">
         <button type="button" class="close" data-dismiss="modal" aria-hidden="true"
-                ng-click="vm.clear()">&times;</button>
+                (click)="clear()">&times;</button>
         <h4 class="modal-title" translate="entity.delete.title">Confirm delete operation</h4>
     </div>
     <div class="modal-body">
         <jhi-alert-error></jhi-alert-error>
-        <p translate="userManagement.delete.question" translate-values="{login: '{{vm.user.login}}'}">Are you sure you want to delete this User?</p>
```

Comment:
No it used in the translation "Are you sure you want to delete user {{ login }}?" so just leave it as it is as it will work with ng2-translate we merged recently